### DISCUSSION TITLE
How to stop zoom animation?

### DISCUSSION THREAD

**Discussion by Xio1996** [Stellarium/stellarium#3072] [Q&A] [answered]
Hi,

I was hoping there was a setting to stop the zoom animation and instead just display the zoomed FOV directly?

The move animation can be stopped by setting the move_speed parameter in the config.ini file to 0. I tried setting the zoom_speed to 0 but it had no effect.

Many thanks for any help.

Pete

**Comment by 10110111** [comment]
Shouldn't a larger value result in reaching the target faster, rather than a lower value?

**Reply by Xio1996** [reply]
Hi,

Thank you for the reply. I tried a higher number 1 and a 1000 and a smaller number 0.001 etc but it still animates the zoom. The move animation is turned off by setting a value of zero so I guessed it might be zero for zoom as well.

Many thanks

Pete

**Reply by 10110111** [reply]
What method of zooming do you use? Scrolling with the mouse?

**Comment by Xio1996** [comment]
Hi,

I use the CTRL+ALT+number. So from a FOV, say zoom 2, I would select CTRL+ALT+9 and expect the FOV of view to just show the zoomed FOV.

Pete

**Comment by 10110111** [comment, marked as answer]
Looks like <kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>digit</kbd> use case is hard coded to 1 second, as can be seen [here](https://github.com/Stellarium/stellarium/blob/master/src/core/StelMovementMgr.cpp#L1027). You can either change the code and rebuild Stellarium, or create an issue asking to implement a control, or even make a pull request that implements it.

**Reply by Xio1996** [reply]
Thank you for taking a look. 

It might be fun to build Stellarium.

Many thanks

Pete
